jetstream/resource_jetstream_consumer.go

```diff
@@ -480,11 +480,11 @@ func resourceConsumerRead(d *schema.ResourceData, m any) error {
 
 	switch cons.DeliverPolicy() {
 	case api.DeliverAll:
-		d.Set("delivery_all", true)
+		d.Set("deliver_all", true)
 	case api.DeliverNew:
-		d.Set("delivery_new", true)
+		d.Set("deliver_new", true)
 	case api.DeliverLast:
-		d.Set("delivery_last", true)
+		d.Set("deliver_last", true)
```

Comment:
doh, well spotted.

Reply:
Hmm, I think these are in fact `all`, `last`, `by_start_sequence`, `by_start_time`, `last_per_subject` and `new`

Reply:
Would also need to fix all these 

https://github.com/nats-io/terraform-provider-jetstream/blob/b43dbe2e9c54fb27881066a1920af0eff70745c9/jetstream/resource_jetstream_consumer.go#L72

Just take those lines out on the deliver policy ones as tehy are all boolean